invenio_matcher/core.py

```diff
@@ -91,6 +90,8 @@ def _get_values(record, match):
     Ensures that the values will be a list, since this is what the
     rest of the code expects.
     """
+    if not isinstance(match, six.text_type):
```

Comment:
FYI: `six.text_type` is `unicode` in Python 2 and `str` in Python 3, because it's the type that you would use to store text.

What you want here is `six.string_types`, which is `basestring` in Python 2 and `str` in Python 3, since is the base type of all things that are strings.